src/operator/convolution_v1.cc

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file convolution_v1.cc
+ * \brief
+ * \author Bing Xu
+*/
+
+#include "./convolution_v1-inl.h"
+#if MXNET_USE_MKL2017 == 1
+#include <mkl_memory.h>
+#include "./mkl/mkl_memory-inl.h"
+#include "./mkl/mkl_convolution-inl.h"
+#endif  // MXNET_USE_MKL2017
+#if MXNET_USE_NNPACK == 1
+#include "./nnpack/nnpack_convolution-inl.h"
+#endif  // MXNET_USE_NNPACK
+
+namespace mxnet {
+namespace op {
+DMLC_REGISTER_PARAMETER(ConvolutionV1Param);
+
+template<>
+Operator* CreateOp<cpu>(ConvolutionV1Param param, int dtype,
+                        std::vector<TShape> *in_shape,
+                        std::vector<TShape> *out_shape,
+                        Context ctx) {
+  Operator *op = NULL;
+#if 0
+#if MXNET_USE_MKL2017 == 1
```

Comment:
There is a problem reverting this. MKLConvolutionOp takes ConvolutionParam as one of its arguments. For ConvolutionV1Op, I had to change ConvolutionParam to ConvolutionV1Param to avoid naming conflict with the ConvolutionParam in the new convolution header file. Reverting this would lead to compile failure. Any suggestions to fix?